Security.sln

```diff
@@ -352,5 +366,6 @@ Global
 		{A7922DD8-09F1-43E4-938B-CC523EA08898} = {4D2B6A51-2F9F-44F5-8131-EA5CAC053652}
 		{A2B5DC39-68D5-4145-A8CC-6AEAB7D33A24} = {7BF11F3A-60B6-4796-B504-579C67FFBA34}
 		{3A7AD414-EBDE-4F92-B307-4E8F19B6117E} = {F8C0AA27-F3FB-4286-8E4C-47EF86B539FF}
+		{B6297507-EECF-4114-9B38-6CCE7F9C3EB5} = {4D2B6A51-2F9F-44F5-8131-EA5CAC053652}
```

Comment:
Make sure to set the solution configuration for the new project to not build it.